src/dataIntegration/metabotools/computeFluxSplits.m

```diff
@@ -14,15 +14,31 @@
 %    mets:      a list of metabolite identifiers from model.mets
 %    V:         an `n` x `k` matrix of `k` flux distributions
 %
+% OPTIONAL INPUT:
+%    coeffSign: only use the sign of the stoichiometric coefficient
+%               (i.e. +1 or -1, Default = false)
+%
 % OUTPUTS:
 %    P:     an `n` x `k` matrix of relative contributions to the production of mets
 %    C:     an `n` x `k` matrix of relative contributions to the consumption of mets
 %    vP:    an `n` x `k` matrix of net producing fluxes
 %    vC:    an `n` x `k` matrix of net consuming fluxes
 %
+% Example: [P,C,vP,vC] = computeFluxSplits(model,mets,V,1);
+%
 % .. Author: - Hulda S. Haraldsdottir, December 2, 2016
+%            - Modified by Diana El Assal 22/7/2017
+
+if nargin <4;
+    coeffSign = 0; %default
+end
+
+if coeffSign == 1;
```

Comment:
When using booleans, it is sufficient to use:
```
if coeffSign
else
end
```
The is no need for `if coeffSign == 0`.